signer/tests/integration/containers.rs

```diff
@@ -0,0 +1,305 @@
+use bitcoin::{AddressType, Amount};
+use bitcoincore_rpc::RpcApi as _;
+use sbtc::testing::{
+    containers::{TestContainers as BaseTestContainers, TestContainersBuilder},
+    regtest::{
+        BITCOIN_CORE_RPC_PASSWORD, BITCOIN_CORE_RPC_USERNAME, BITCOIN_CORE_WALLET_NAME,
+        FAUCET_LABEL, FAUCET_SECRET_KEY, Faucet, MIN_BLOCKCHAIN_HEIGHT, get_or_create_wallet,
+    },
+};
+use signer::{
+    bitcoin::rpc::BitcoinCoreClient, error::Error, storage::postgres::PgStore,
+    testing::storage::get_connection_pool,
+};
+use tokio::sync::OnceCell;
+use url::Url;
+
+const COMPOSE_BITCOIN: &str = "docker-compose.bitcoin.yml";
+const COMPOSE_DATABASE: &str = "docker-compose.db.yml";
+const COMPOSE_EMILY: &str = "docker-compose.emily.yml";
+
+const SERVICE_BITCOIN: &str = "bitcoind";
+const SERVICE_BITCOIN_RPC_PORT: u16 = 18443;
+
+const SERVICE_DATABASE: &str = "postgres";
+const SERVICE_DATABASE_PORT: u16 = 5432;
+const SERVICE_DATABASE_USER: &str = "postgres";
+const SERVICE_DATABASE_PASSWORD: &str = "postgres";
+
+fn compose_path(file_name: &str) -> String {
+    // Evaluate `CARGO_MANIFEST_DIR` at runtime for nextest relocation
+    format!(
+        "{}/../docker/tests/{file_name}",
+        std::env::var("CARGO_MANIFEST_DIR").unwrap()
```

Comment:
This is fine, but I believe we can just use std::env!("CARGO_MANIFEST_DIR") since cargo always sets that environment variable.